Resources/Prototypes/Entities/Objects/Tools/handheld_mass_scanner.yml

```
@@ -61,3 +61,8 @@
         startingItem: PowerCellMicroreactor
         disableEject: true
         swap: false
+
+- type: entity
+  id: HandHeldMassScannerUnpowered
+  parent: HandHeldMassScanner
+  suffix: Handheld, Always Powered
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)